### DISCUSSION TITLE
error message: .quant file was obtained using a different spectral library

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#344] [Q&A]
I ran a DIA-NN 1.8 GUI search with approx 2300 diaPASEF raw files and an xls library from Spectronaut and got the following error message immediately in Cross-run analysis: 

ERROR: a .quant file was obtained using a different spectral library / different library-free settings. 

DIA-NN then exits and does not write any table or report (even though it states 'PDF report will be generated in the background'). 

I have used this library and a smaller set (96) of these raw files before with the same settings and the search ran through without problems. 

Q1: What causes this error?
Q2: How can it be fixed/prevented?

![image](https://user-images.githubusercontent.com/98888664/159007409-acb47502-9e59-4422-b393-54b4b6a3061f.png)

**Comment by vdemichev** [comment]
Most likely: an analysis was run in parallel on one of the runs in this experiment. This overwrote one or several of the .quant files. Or can be that just a .quant file got corrupted on the disk. So the options are to rerun the whole thing, or, of this is too long, try to figure out which .quant file(s) is responsible and just regenerate specifically those, and then summarise the whole experiment using the 'Use existing .quant files' option.

**Comment by vdemichev** [comment]
Prevention: better to always specify a Temp/.dia dir, where .quant files will be saved.
